KNOWN_ISSUES.md

```diff
@@ -136,6 +137,34 @@ kubectl get --raw /api/v1/nodes/$(kubectl get nodes -o json  | jq -r '.items[0].
 
   When launching minikube set `kubelet.housekeeping-interval`, for example `minikube start --extra-config=kubelet.housekeeping-interval=10s`
 
+## Metrics-server pod failed to reach running status
+
+**Symptoms**
+
+When running `kubectl get pods -n kube-system` and not get running status for the metrics-server pod.
+
+```
+NAME                                           READY   STATUS    RESTARTS   AGE
+metrics-server-dbf765b9b-mhqm7                 0/1     Running   0          11m
+```
+
+**Debugging**
+
+Please check if your metrics server reports problems to scrape node, in particular errors will include
+```
+"Failed to scrape node" err="Get "https://192.168.65.4:10250/metrics/resource\": context deadline exceeded" node="docker-desktop"
+"Failed probe" probe="metric-storage-ready" err="no metrics to serve"
+```
+
+**Known Causes**
+
+metrics-server scrape kubelet `/metrics/resource` endpoint timeout.
+
+**Workaround**
+
+* The main branch can be resolved by adjusting the `--kubelet-request-timeout` flag.
```

Comment:
Don't agree that it is a good recommendation. Timeouts on node scrape is a symptom of a bigger problem with networking. Increasing the timeout will not help.

Reply:
I think  I dealt with it too simply. I have updated. , please take a look again, thanks.